library/alloc/src/collections/vec_deque/mod.rs

```diff
@@ -2435,7 +2433,7 @@ impl<T> VecDeque<T> {
     /// deque.insert(idx, num);
     /// assert_eq!(deque, &[0, 1, 1, 1, 1, 2, 3, 5, 8, 13, 21, 34, 42, 55]);
     /// ```
-    #[unstable(feature = "vecdeque_binary_search", issue = "78021")]
+    #[stable(feature = "vecdeque_binary_search", since = "1.52.0")]
```

Comment:
The version was just bumped to 1.53: https://github.com/rust-lang/rust/pull/83324
```suggestion
    #[stable(feature = "vecdeque_binary_search", since = "1.53.0")]
```